openpype/hosts/tvpaint/plugins/create/create_render.py

```diff
@@ -604,6 +607,356 @@ def get_instance_attr_defs(self):
         return self.get_pre_create_attr_defs()
 
 
+class TVPaintAutoDetectRenderCreator(TVPaintCreator):
+    """Create Render Layer and Render Pass instances based on scene data.
+
+    This is auto-detection creator which can be triggered by user to create
+    instances based on information in scene. Each used color group in scene
+    will be created as Render Layer where group name is used as variant and
+    each TVPaint layer as Render Pass where layer name is used as variant.
+
+    Never will have any instances, all instances belong to different creators.
+    """
+
+    family = "render"
+    label = "Auto detect renders"
```

Comment:
```suggestion
    label = "Render Layer/Passes"
```
This would need to be renamed throughout if you think its a good name.

Reply:
I agree it should be renamed. Don't know if the suggested is good.

Reply:
I just realized I didn't add description for user. So the label could be enough if there would be the description.